bitrise.yml

```diff
@@ -1,84 +1,27 @@
-format_version: 8
+format_version: "11"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
-app:
-  envs:
-  # define these in your .bitrise.secrets.yml
-  - INVALID_SSH_KEY: $INVALID_SSH_KEY
-  - MISSING_NEWLINE_SSH_KEY: $MISSING_NEWLINE_SSH_KEY
-  - PEM_FORMAT_SSH_KEY: $PEM_FORMAT_SSH_KEY
-  - OPENSSH_FORMAT_SSH_KEY: $OPENSSH_FORMAT_SSH_KEY
-
 workflows:
-  ci:
-    before_run:
-    - audit-this-step
+  generate-readme:
```

Comment:
I think it should be `generate_readme` to match our conventions. (Sorry for not noticing earlier 🙈)